torch/_dynamo/variables/misc.py

```diff
@@ -1188,16 +1247,16 @@ def call_method(
 
 
 class LambdaVariable(VariableTracker):
-    def __init__(self, fn, **kwargs) -> None:
+    def __init__(self, fn: Callable[..., VariableTracker], **kwargs: Any) -> None:
```

Comment:
Feels like the Callable fn here needs to take VariableTracker is positional args and valietract divtionary as kwargs. May want to enforce or better yet store a ParamSpec generic here to handle it properly later in call_function

Reply:
Left todo to specify this is variable number of VariableTrackers in the fn input; we can do that with TypeVarTuple which is available in 3.11+